doc/book/admin/upgrades.rst

```diff
@@ -58,95 +25,113 @@ How to upgrade from Tarantool 1.7 to 2.x
 3. Update the Tarantool server. See installation instructions at Tarantool
    `download page <http://tarantool.org/download.html>`_.
 
-4. Launch the updated Tarantool server using ``tarantoolctl`` or ``systemctl``.
+After that, make sure to :ref:`finish the upgrade properly <admin-upgrades_db>`.
 
-.. _admin-upgrades_instance_16_to_20:
+.. _admin-upgrades_replication_cluster:
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-How to upgrade from Tarantool 1.6 to 2.x
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Upgrading Tarantool in a replica set with no downtime
+-----------------------------------------------------
 
-The procedure is fully analogous to
-:ref:`upgrading from 1.7 to 2.x <admin-upgrades_instance_17_to_20>`.
+Below are the general instructions for upgrading Tarantool in a replica set.
+Upgrading from some versions can involve certain specifics. You can find
+instructions for individual versions :ref:`in the list below <admin-upgrades_version_specifics>`.
 
-.. _admin-upgrades_instance_16_to_17:
+..  important::
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-How to upgrade from Tarantool 1.6 to 1.7
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    The only way to upgrade Tarantool from version 1.6, 1.7, or 1.9 to 2.x **without downtime** is
+    taking an intermediate step by upgrading to 1.10 and then to 2.x.
 
-This procedure is for upgrading a standalone Tarantool instance in production
-from 1.6.x to 1.7.x. Notice that this will **always imply a downtime**.
-To upgrade **without downtime**, you need several Tarantool servers running in a
-replication cluster (see :ref:`below <admin-upgrades_replication_cluster>`).
+    Before upgrading Tarantool from 1.6 to 2.x, please read about the associated
+    :ref:`caveats <admin-upgrades-1.6-1.10>`.
 
-Tarantool 1.7 has an incompatible :ref:`.snap <internals-snapshot>` and
-:ref:`.xlog <internals-wal>` file format: 1.6 files are
-supported during upgrade, but you won’t be able to return to 1.6 after running
-under 1.7 for a while. It also renames a few configuration parameters, but old
```

Comment:
Please add this part to "caveats when upgrading 1.6 to 2.x"

Saying that tarantool 1.10 (and 2.x) have incompatible snapshot format compared to 1.6
So the user won't be able to return to 1.6 after running 1.10 or 2.x for a while